client/dashboard/sites/overview-visibility-card/index.tsx

```diff
@@ -117,6 +118,10 @@ function VisibilityCardPublic( { site }: { site: Site } ) {
 }
 
 export default function VisibilityCard( { site }: { site: Site } ) {
+	if ( ! canViewSiteVisibilitySettings( site ) ) {
```

Comment:
I think this check is redundant? We already check for it in the root Overview screen.

Reply:
It does sound redundant, yes.